templates/web/peterborough/email_sent.html

```diff
@@ -27,17 +27,9 @@ <h1>[% loc("Nearly done! Now check your email&hellip;") %]</h1>
 
     <p>
     [% IF email_type == 'problem' %]
-        [% IF non_public %]
```

Comment:
Thanks for this, but I think you could take all of this from the h1 to the "Can't find our email" p into one template - it's identical here (and then not identical for TfL) :)